spec/shared_examples_for_proofers.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+RSpec.shared_examples 'misconfigured proofer' do
+  it 'raises error if auth token is not configured and block is not given' do
```

Comment:
should we add in a `before { ENV.delete('IDP_API_AUTH_TOKEN') }` just to be sure?

Reply:
I did `ENV['IDP_API_AUTH_TOKEN'] = nil` to avoid invoking SSM